src/main/java/de/qabel/desktop/ui/contact/context/AssignContactController.java

```diff
@@ -63,6 +73,18 @@ public void initialize(URL location, ResourceBundle resources) {
         addIgnoreButton(resources);
     }
 
+    String formatKey(String keyIdentifier) {
+        char[] chars = keyIdentifier.toCharArray();
+        StringBuilder formatted = new StringBuilder();
+        for (int i = 0; i < chars.length; i++) {
+            if (i > 0 && i%4 == 0) {
+                formatted.append(" ");
+            }
+            formatted.append(chars[i]);
+        }
+        return formatted.toString();
+    }
```

Comment:
I think we have a method in the core for that. Maybe in the form of an extension method, so you maybe have to wrap it.